ml/src/main.py

```diff
@@ -1,13 +1,31 @@
 from flask import Flask
 import os
+from sentence_transformers import SentenceTransformer, util
+from PIL import Image
 
+model = SentenceTransformer('clip-ViT-B-32')
 server = Flask(__name__)
 is_dev = os.getenv('NODE_ENV') == 'development'
 
 
 @server.route("/")
 def hello():
-    return "Immich Machine Learning!\n"
+    print("Triggering hello route", flush=True)
+    return "Immich Machine Learning 12!\n"
+
+
+@server.route("/test")
+def test():
+    img_emb = model.encode(Image.open(
+        'upload/3e54a230-5daa-4983-b356-af4fd65292aa/thumb/WEB/bf7d976f-4cbc-4991-a006-b11c0811781c.jpeg'))
```

Comment:
I think we should be OK to just point at the original and sbert will resize internally. Not sure if it supports all the file formats though. Do we have full-size jpeg/webp or only thumbs?

Reply:
(I know you're just experimenting ATM, but thought this was worth noting anyways)